### PR TITLE
State Management curated page

### PR DESCRIPTION
**Pull request by laurosilvacom**
## [Live Preview](https://egghead-io-nextjs-git-ls-state-management-eggheadio1.vercel.app/learn/state-management)

CleanShot 2021-07-26 at 08 43 01@2x

### CODE REVIEW

File: src/pages/learn/state-management/index.tsx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    title,
+  	description,
+  	"name": subTitle
+  },
+  'theBigIdeas': resources[slug.current == 'the-big-ideas'][0]{
+    description,
+    title,
+    resources[] {
+    "name": subTitle,
+      ...
```
Comment (zacjones93) [inline src/pages/learn/state-management/index.tsx:49]: oh nice, didn't know you could spread like this

File: src/pages/learn/state-management/index.tsx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+      </div>
+    </div>
+  )
+}
+
+type JumbotronProps = {
+  resource: CardResource
+}
+
+const Jumbotron: FunctionComponent<JumbotronProps> = ({resource}) => {
```
Comment (zacjones93) [inline src/pages/learn/state-management/index.tsx:139]: Is there a reason we define another Jumbotron here vs using the on in `/components`?

File: src/pages/learn/state-management/index.tsx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+  const data = await sanityClient.fetch(StateManagementQuery)
+
+  return {
+    props: {
+      data,
+    },
+  }
+}
+
+const RecoilSection = ({resource}: any) => {
```
Comment (zacjones93) [inline src/pages/learn/state-management/index.tsx:70]: These sections are great candidates to promote to a shared components folder

**Review by zacjones93 — APPROVED** [review]
This looks like a really good start to the state management page. Lots of room to build on!

The recoil section could use some gardening on telling us what problems it solves that React isn't good at but that can be done on the content side of things

File: src/pages/learn/state-management/index.tsx (hunk 2 above)
Comment (laurosilvacom) [reply src/pages/learn/state-management/index.tsx:139]: Had a couple of issues passing the appropriate values. But that's definitely something to refactor the next time we work on this page.

**Pull request merged**